libs/datasets/sources/nytimes_dataset.py

```diff
@@ -47,11 +47,31 @@ def local(cls) -> "NYTimesDataset":
     @classmethod
     def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         data[cls.Fields.COUNTRY] = "USA"
+        data[cls.Fields.AGGREGATE_LEVEL] = "county"
         data = dataset_utils.strip_whitespace(data)
         data[cls.Fields.STATE] = data[cls.Fields.STATE].apply(dataset_utils.parse_state)
         # Super hacky way of filling in new york.
         data.loc[data[cls.Fields.COUNTY] == "New York City", "county"] = "New York County"
         data.loc[data[cls.Fields.COUNTY] == "New York County", "fips"] = "36061"
         data.loc[data[cls.Fields.COUNTY] == "Unknown", "fips"] = "99999"
-        data[cls.Fields.AGGREGATE_LEVEL] = "county"
-        return data
+
+        # https://github.com/nytimes/covid-19-data/blob/master/README.md#geographic-exceptions
+        # Both Joplin and Kansas City numbers are reported separately from the surrounding counties.
+        # Until we figure out a better way to spread amongst counties they are in, combining all
```

Comment:
If we don't think NYT is going to ever fix this, should we just apportion based on relative population of the counties and call it good enough. I don't see an obviously better solution that doesn't get complicated or brittle quickly.

Reply:
Yeah I went back and forth on what the best way to handle that is, but ultimately I think it might worth a trello task to dig a bit more into it.  

I do agree that population seems reasonable but I'm not sure at what point that begins to break down - also for a while they counted some of these in the Unknown county, which is no longer around. 

I'm not totally opposed to fixing it here though if you want